core/db_models/EEM_Payment_Method.model.php

```diff
@@ -111,112 +174,129 @@ public function is_valid_scope($scope)
         $scopes = $this->scopes();
         if (isset($scopes[ $scope ])) {
             return true;
-        } else {
-            return false;
         }
+        return false;
     }
 
 
-
     /**
      * Gets all active payment methods
+     *
      * @param string $scope one of
      * @param array  $query_params
+     * @return EE_Base_Class[]|EE_Payment_Method[]
      * @throws EE_Error
-     * @return EE_Payment_Method[]
      */
-    public function get_all_active($scope = null, $query_params = array())
+    public function get_all_active($scope = null, $query_params = [])
     {
         if (! isset($query_params['order_by']) && ! isset($query_params['order'])) {
-            $query_params['order_by'] = array( 'PMD_order' => 'ASC', 'PMD_ID' => 'ASC' );
+            $query_params['order_by'] = ['PMD_order' => 'ASC', 'PMD_ID' => 'ASC'];
         }
         return $this->get_all($this->_get_query_params_for_all_active($scope, $query_params));
     }
 
+
     /**
      * Counts all active gateways in the specified scope
+     *
      * @param string $scope one of EEM_Payment_Method::scope_*
-     * @param array $query_params
+     * @param array  $query_params
      * @return int
+     * @throws EE_Error
+     * @throws EE_Error
```

Comment:
Looks like these doubled. Or one of these had to be of a different error type ?
```suggestion
     * @throws EE_Error
```

Reply:
yup... magical IDE updates not turning out well

thnx will update